mobly/controllers/android_device_lib/jsonrpc_client_base.py

```diff
@@ -385,7 +390,7 @@ def _id_counter(self):
             yield i
             i += 1
 
-    def set_snippet_client_verbose_logging(self, verbose=False):
+    def set_snippet_client_verbose_logging(self, verbose=True):
```

Comment:
i don't think we need a default value for `verbose`
since this is the setter function, which does not determine the default behavior, right?